### PR TITLE
Add Work Section

### PR DESCRIPTION
**Pull request by richy4dev**
In this branch, I added a work section with HTML and CSS file

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -84,5 +84,282 @@ <h3>LET’S CONNECT</h3>
...
+                  <img
+                    class="lang-icon"
+                    src="Image/HTML.png"
+                    alt="HTML icon"
+                  />
+                  <span class="lang-name">HTML</span>
+                </li>
+                <li class="flex view-list-item">
+                  <img
+                    class="lang-icon"
```
Comment (tufoinnkuo10) [inline Index.html:310]: - [ ] kindly remove the dark background and follow the figam design. 
your version
Screenshot (203)
figma version

Screenshot (204)

File: Index.html [hunk 2]
```diff
@@ -84,5 +84,282 @@ <h3>LET’S CONNECT</h3>
...
+            sign-ups required.
+          </p>
+          <ul class="work-categories margin-top-12 padding-0">
+            <li><span>html</span></li>
+            <li><span>css</span></li>
+            <li><span>javascript</span></li>
+          </ul>
+          <div>
+            <button type="button" class="button button-enabled">
+            See Project
```
Comment (tufoinnkuo10) [inline Index.html:173]: - [ ] kindly align the see project button with the card text so it follows the figma design.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @richy4dev 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅figma design followed

## Required Changes ♻️♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Kossi-stack** [comment]
Stay online on Slack whenever you submit a PR

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @richy4dev,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).
 - [ ] Kindly align see project button and About Myself well


 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by vickymarz — APPROVED** [review]
Hi @richy4dev , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by richy4dev** [comment]
Thank you all for your reviews. much appreciated.

**Pull request closed without merging**